fixtures/abl_books_uuids_slugs.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+"""
+Creates a dictionary of book uuid:slug for all github content repos.
+Latest update on April 29th, 2022
+"""
+
+
+@pytest.fixture
+def abl_books_uuids_slugs(abl_approved):
+
+    slugs = []
+    uuids = []
+
+    for i in abl_approved:
+        book_versions = i["versions"]
+        for j in book_versions:
+            commit_metadata = j["commit_metadata"]["books"]
+            for k in commit_metadata:
+                slugs.append(k["slug"])
+
+    for i in abl_approved:
+        book_versions = i["versions"]
+        for j in book_versions:
+            commit_metadata = j["commit_metadata"]["books"]
+            for k in commit_metadata:
+                uuids.append(k["uuid"])
+
```

Comment:
Seems like this could be rephrased to
```python
uuids_slugs = {}

for i in abl_approved:
    book_versions = i["versions"]
    for j in book_versions:
        commit_metadata = j["commit_metadata"]["books"]
        for k in commit_metadata:
            uuids_slugs[k["uuid"]] = k["slug"]
```
if you wanted to avoid looping three times.

Reply:
yes, thanks :)